### PR TITLE
Implemented first version of notifications

### PR DESCRIPTION
**Pull request by cyterdan**
Added internationalization for notification messages

### CODE REVIEW

File: db_schemas/sandbox.sql [hunk 1]
```diff
@@ -1,28 +1,44 @@
...
 
 truncate contact_tracer.device_location ;
 INSERT INTO contact_tracer.device_location (device_id,sample_date,location) VALUES 
-('device1', now(), Point(29.733521,-95.388815)),
-('device1', now() - interval '2 hours', Point(29.733730, -95.387850)),
-('device1', now() - interval '3 hours', Point(29.734117, -95.387850)),
-('device1', now() - interval '4 hours', Point(29.734410, -95.388113)),
-('device2', now() - interval '2 hours', Point(29.7344752,-95.3882035)),
-('device3', now() - interval '3 days', Point(29.7344752,-95.3882035));
+('device1', now(), ST_MakePoint(29.733521,-95.388815)),
```
Comment (jmrr) [inline db_schemas/sandbox.sql:14]: I had to look up this function, very handy :+1:

File: api/handler.py [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
+i18n.load_path.append('./config/translations')
+
+
+def notifyDevice(deviceId, title, body):
+    print(f"notifying {deviceId} with {title}|{body}")
+    response = requests.post(EXPO_PUSH_ENDPOINT, json={'to': deviceId, 'title': title, 'body': body})
+    print(response.content)
+
 
 def make_conn():
```
Comment (jmrr) [inline api/handler.py:24]: Thanks for being on the ball, I built this one so when we merge it'll be usable.

File: api/handler.py [hunk 3]
```diff
@@ -36,16 +48,45 @@ def postLocation(event, context):
...
+    cursor.execute(
+        f"update contact_tracer.device set infected = true, infected_date = '{event['infectionDate']}' where device_id = '{event['deviceId']}'; ")
+    # get all devices that need to be notified
+    cursor.execute(f"""
+    select distinct dl3.device_id as notifyDeviceId
+    from 
+    contact_tracer.device_location dl 
+    join lateral 
+    (
+        select * from contact_tracer.device_location where device_id != dl.device_id and ST_DistanceSphere(location,dl.location) < 500
```
Comment (jmrr) [inline api/handler.py:65]: Maybe we should use the second parameter of `execute` so we can claim that we're robust against sql injection? But perhaps do this in a second iteration and work on more functionality.

**Review by jmrr — APPROVED** [review]
Just left some comments for a second iteration seeking robustness, let me know what you think.

File: api/handler.py (hunk 3 above)
Comment (cyterdan) [reply api/handler.py:65]: good point

**Pull request merged**